code/obj/decal/cleanable.dm

```diff
@@ -759,10 +759,6 @@ var/list/blood_decal_violent_icon_states = list("floor1", "floor2", "floor3", "f
 	var/artist = null//the key of the one who wrote it
 	real_name = "writing"
 
-	New()
-		. = ..()
-		src.create_reagents(PEN_REAGENT_CAPACITY)
```

Comment:
don't we need this?

Reply:
I don't think so? It was added for the pen reagent writing stuff, but cleanables use a different system so it just caused bugs.